packages/muelu/research/mmayr/composite_to_regions/src/main.cpp

```diff
@@ -935,7 +934,7 @@ int main_(int argc, char *argv[]) {
       /////////////////////////////////////////////////////////////////////////
 
 //      printRegionalObject<Vector>("regB 2", regB, myRank, *fos);
-      vCycle(0, numLevels, maxCoarseIter, maxRegPerProc,
```

Comment:
After this change `const int maxCoarseIter = 100;` on line 891 becomes unused and needs to be removed (-Werror does not like it)